lib/features/login/domain/state/get_stored_token_oidc_state.dart

```diff
@@ -1,19 +1,20 @@
 import 'package:core/presentation/state/failure.dart';
 import 'package:core/presentation/state/success.dart';
-import 'package:model/oidc/token_oidc.dart';
+import 'package:model/model.dart';
```

Comment:
I think, we should import exactly what we use, it is optimized for our code.

Reply:
We are using multiple classes in this module. I don't want to write too much code for it. This is the best way for us in the future.

Reply:
Ok, we will only `import` the `classes` that we use.